config/default.schema.json

```diff
@@ -966,6 +966,11 @@
 			"default": false,
 			"description": "Toggle for lesson sharing feature."
 		},
+		"FEATURE_LESSON_SHARE_NEW": {
```

Comment:
this reminds me a bit of Windows NT ("new technology"... a windows version that is very, very old now).
I imagine changing this would be quite some effort now requiring changes in a multitude of repositories.

At the very least please create a ticket to change this once sharing is done and the old feature is gone.
(remove all code behind the old variable, and rename this variable to FEATURE_LESSON_SHARE). Please talk to ingwert and ask when the old feature will be disabled everywhere (currently only active in nbc afaik)

Reply:
Done https://ticketsystem.dbildungscloud.de/browse/BC-3197